### ISSUE TITLE
Present Awards -  Feature Request - Date for presented

### ISSUE THREAD

**Issue by AceLeader2023** [pete-mc/Summit#28]
Is it possible that we could have a date when an award is set to be presented on the present awards tool???

**Comment by pete-mc** [comment]
Yeah this one is a bit tricky, I am using a calendar entry to store the data for the presentation and need to try to keep the number of characters down for each award to stay withing field size limits (ideally 1000 characters per youth member) as I am using an activity per youth member in the event and the description field is 1000. I am not sure if there is a limit to the number of activities per event.

Currently if a youth member exceeds 1000 characters it creates an extra activity to keep storing more.

The award id is a guid like this: e3063def-dd74-46a7-8232-65d3f6a7d56d I managed to compress it down to 16 characters by removing the dashes and going from base 16 to base 300. I might be able to do a similar trick with a 6 digit date to get it down to 3 which will be 19 characters per award. maybe that is not as bad as I was thinking.

That would take it from about 62 badges per activity stored down to 52 so that is probably an ok compromise. 

I will sleep on it 😄

**Comment by pete-mc** [comment]
I will move this to the 2.5.0 sprint so I have a bit more time to think about it. I realised I am going to have to work out a way to upgrade the old version so people don't lose their data.

**Comment by pete-mc** [comment]
Just an update to let you know this is almost complete, just have an annoying bug with the date picker. Should be released in next release. Sorry for the delay in this one.

**Comment by pete-mc** [comment]
We have decided to remove the presented feature from Summit as this is now supported directly within Terrain.
